projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/description/interestrate/NormalSTIRFuturesProviderInterface.java

```diff
@@ -11,28 +11,28 @@
  * Provider of normal volatility (Bachelier model) smile for options on STIR futures. The volatility is time to expiration/strike/delay dependent. 
  * The "delay" is the time between expiration of the option and last trading date of the underlying futures.
  */
-public interface NormalSTIRFuturesSmileProviderInterface extends ParameterProviderInterface {
+public interface NormalSTIRFuturesProviderInterface extends ParameterProviderInterface {
 
   /**
    * Create a new copy of the provider
    * @return The bundle
    */
   @Override
-  NormalSTIRFuturesSmileProviderInterface copy();
+  NormalSTIRFuturesProviderInterface copy();
 
   /**
    * Gets the normal volatility at a given expiry-strike-delay point.
    * @param expiry The time to expiration.
-   * @param strike The strike.
    * @param delay The delay between expiration of the option and last trading date of the underlying futures.
+   * @param strike The strike price.
+   * @param futuresPrice The price of the underlying futures. Used for relative moneyness smile description.
    * @return The volatility.
    */
-  double getVolatility(final double expiry, final double strike, final double delay);
+  double getVolatility(final double expiry, final double delay, final double strike, final double futuresPrice);
 
   /**
    * Returns the Ibor Index of the futures on for which the Black data is valid, i.e. the data is calibrated to futures on the given index.
    * @return The generator.
    */
   IborIndex getFuturesIndex();
-
 }
```

Comment:
To facilitate the Finite Difference curve sensitivity code, we may need to add a "withMulticurve" method in the interface. The method would create another instance of the (normal) data but with a different multicurve provider.